### PR TITLE
Update events in doc

### PR DESCRIPTION
**Pull request by babsey**

### CODE REVIEW

File: docs/about/events.rst [hunk 1]
```diff
@@ -1,10 +1,21 @@
 Events
 ======
 
+2023
+----
+
+:13 - 24 Feb: BSc course "Simple Neuron Models" at BCF [#f1]_, Freiburg, Germany.
+:18 Jan: Part of the talk at CENIA, presented by Markus Diesmann, Santiago, Chile.
+:18 - 20 Jan: Poster at `7th HBP Student Conference <https://www.humanbrainproject.eu/en/education-training-career/HBPSC2023/>`, Madrid, Spain (cancelled).
```
Comment (jens-br) [inline docs/about/events.rst:9]: I do have first-hand information that it was not cancelled!!! :joy: 
```suggestion
:18 - 20 Jan: Poster at `7th HBP Student Conference <https://www.humanbrainproject.eu/en/education-training-career/HBPSC2023/>`__, Madrid, Spain.
```

**Review by jens-br — CHANGES_REQUESTED** [review]
Seems good, except for one line.

File: docs/about/events.rst [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
+
 2022
 ----
 
+:07 - 10 Nov: Session talk "NEST Desktop" at `Simulate with EBRAINS <https://flagship.kip.uni-heidelberg.de/jss/HBPm?m=showAgenda&meetingID=242>`__, online.
+:13 - 16 Sep: Poster "NEST Desktop: Explore new frontiers" at `Bernstein Conference 2021 <https://abstracts.g-node.org/conference/BC22/abstracts#/uuid/b205c368-bbfa-473f-a249-eb866c9fdffc>`__, Berlin, Germany.
+:20 - 21 Jul: Talk "NEST Desktop" at HCI Summer Colloquium, Trier, Germany.
 :16 Jul: NEST Desktop is part of the onsite tutorial "T1: From single-cell modeling to large-scale network dynamics with NEST Simulator." of `CNS 2022 <https://www.cnsorg.org/cns-2022-tutorials>`__, presented by Jasper Albers, Pooja Babu and Charl Linssen, Melbourne, Australia.
+:03 - 09 Jul: Part of the workshop "NEST Workshop: network and plasticity" at `9th BNNI <http://bionn.matinf.uj.edu.pl/events/bnni2022/#program>`, presented by Jasper Albers, Krakau, Poland.
```
Comment (jens-br) [inline docs/about/events.rst:18]:
```suggestion
:03 - 09 Jul: Part of the workshop "NEST Workshop: network and plasticity" at `9th BNNI <http://bionn.matinf.uj.edu.pl/events/bnni2022/#program>`__, presented by Jasper Albers, Krakau, Poland.
```

**Review by jens-br — CHANGES_REQUESTED** [review]
*two lines, sry

**Comment by babsey** [comment]
We will come back to this style issue later.

**Review by jens-br — APPROVED** [review]
LGTM :+1:

**Pull request merged**